### PR TITLE
Add comment about releasing new modules

### LINKED ISSUE #804: Add javadoc bundling step in RELEASING docs
This is pertinent when releasing a new module.  An example PR is #802 for DNS.

### PR DESCRIPTION
**Pull request by ajkannan**
Fixes #804

### CODE REVIEW

**Comment by mziccard** [comment]
LGTM

**Comment by ajkannan** [comment]
Added a detail to the javadoc grouping in c6c697, will squash and merge after approved.

File: RELEASING.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Most of the release process is handled by the `after_success.sh` script, trigger
 1. Run `utilities/update_pom_version.sh` from the repository's base directory.
 This script takes an optional argument denoting the new version.  By default, if the current version is X.Y.Z-SNAPSHOT, the script will update the version in all the pom.xml files to X.Y.Z.  If desired, another version can be supplied via command line argument instead.
 
-2. Create a PR to update the pom.xml version.  If releasing a new client library, this PR should also update javadoc grouping.
+2. Create a PR to update the pom.xml version.  If releasing a new client library, this PR should also update javadoc grouping in the base directory's pom.xml.
```
Comment (mziccard) [inline RELEASING.md]: You cloud also link it: `[pom.xml](./pom.xml)`

Comment (ajkannan) [inline RELEASING.md]: Link added

**Comment by mziccard** [comment]
Looks good to me, feel free to merge after squashing!

**Pull request merged**